packages/es-components/src/components/util/useTheme.ts

```diff
@@ -1,4 +1,4 @@
 import { useContext } from 'react';
 import { ThemeContext } from 'styled-components';
 
-export const useTheme = () => useContext(ThemeContext);
+export const useTheme = (): Theme => useContext(ThemeContext);
```

Comment:
Just for the sake of clarity, we should import `DefaultTheme` from styled-components and use that.

```suggestion
export const useTheme = (): DefaultTheme => useContext(ThemeContext);
```